share/mk/bsd.cpu.mk

```diff
@@ -484,3 +480,23 @@ MACHINE_ABI+=	ptr64
 .else
 MACHINE_ABI+=	ptr32
 .endif
+
+.if ${MACHINE_ABI:Mpurecap}
+.ifdef CHERI_USE_CAP_TABLE
+FLAGS+=	-cheri-cap-table-abi=${CHERI_USE_CAP_TABLE}
```

Comment:
```suggestion
CFLAGS+=	-cheri-cap-table-abi=${CHERI_USE_CAP_TABLE}
```

Reply:
Yeah, I found that right after I pushed it. :(